docs/designs/xref.md

```diff
@@ -91,8 +91,35 @@ xref:
   - http://url1/.xrefmap.json
   - http://url2/.xrefmap.json
 ```
+User should be able to set a string instead of array, docfx will expand it to an array during the build.\
 During `docfx restore`, all the JSON files will be restored and merged.
 
+#### Xref base path
+Since we do not have a clear picture how we are going to support the local build, we will focus on OPS build firstly.\
+The user needs to set where to get the `.xrefmap.json` files in `.openpublishing.publish.config.json`.
+```json
+{
+    "docsets_to_publish": [
```

Comment:
why are we using ops config? is this a workaround or final design?

Reply:
This config in v2 also belongs to OPS config.
And as it describes, we are focusing on docs build for now, not local build.